library/core/src/char/methods.rs

```diff
@@ -517,8 +518,9 @@ impl char {
     /// assert_eq!('"'.escape_default().to_string(), "\\\"");
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[rustc_const_stable(feature = "const_char_escape_default", since = "1.50.0")]
     #[inline]
-    pub fn escape_default(self) -> EscapeDefault {
+    pub const fn escape_default(self) -> EscapeDefault {
```

Comment:
same here. I think we should not make functions `const fn` for now unless their result can actually be used for anything during const eval or there's a good reason to have a constant version of it

Reply:
This concern seems to still be outstanding.